cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/survey/results-3-5.html

```diff
@@ -0,0 +1,160 @@
+{% extends 'content-base.html' %}
```

Comment:
Similar to my comment on the before-page templates, why are all the result pages duplicated?

Reply:
We expect the content to differ. Once we analyze the full content we'll consolidate if we can.